src/sentry/integrations/vsts/notify_action.py

```diff
@@ -34,5 +35,34 @@ def __init__(self, *args, **kwargs):
     def render_label(self):
         return self.label.format(name=self.get_integration_name())
 
+    def build_description(self, event, installation):
```

Comment:
@ceorourke I think we should subclass `IntegrationEventAction` to `TicketEventAction` then add `build_description` to that class since it's the same for Jira as well.

Reply:
@scefali I was thinking that too - it's not exactly the same but I can probably still pull that off (this one adds a newline char and formats the url differently, [text][url] vs [text|url] ).

Reply:
@ceorourke interesting, why is it different? Either way, you can always make a function that's called here and have it behave differently for each integration.

Reply:
@scefali just different integration behavior. If I added a newline in Jira it came out all wonky and if I didn't add one for Azure the comment was on the same line as the description. For the URL it's just how each party chooses to format it unfortunately.  But yeah I'll work on that.